loom/script/reflection/lsMemberInfo.h

```diff
@@ -384,6 +387,16 @@ class MemberInfo {
         this->type = type;
     }
 
+    inline bool getMissing()
+    {
+        return missing;
+    }
+
+    inline void setMissing(const char *message)
```

Comment:
Actually, I don't know why it's `inline`. Probably because the surrounding functions are as well and I just adapted to the surroundings :large_blue_circle: :large_blue_circle: :large_blue_diamond:  :large_blue_circle: . I can remove it. I think a lot of compilers are pretty soft on the keyword anyway.